packages/network/src/config.ts

```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
 
 export type ChainIdLike = NetworkConfig | BigNumberish
 
-export type NetworksBuilder = (vars: { [key: string]: any }) => NetworkConfig[]
-
-export const mainnetNetworks = createNetworkConfig(
-  (vars: { [key: string]: any }) => [
-    {
-      ...networks[ChainId.MAINNET],
-      ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
-      rpcUrl: urlClean(`${vars.baseRpcUrl}/mainnet`),
-      relayer: { url: urlClean(`${vars.baseRelayerUrl}/mainnet`) },
-      isDefaultChain: true
-    },
-    {
-      ...networks[ChainId.POLYGON],
-      rpcUrl: 'https://rpc-mainnet.matic.network',
-      relayer: { url: urlClean(`${vars.baseRelayerUrl}/matic`) },
-      isAuthChain: true
-    }
-  ],
-  1,
+export const mainnetNetworks = [
+  {
+    ...networks[ChainId.MAINNET],
+    rpcUrl: nodesURL('mainnet'),
+    relayer: { url: relayerURL('mainnet') },
+    indexerUrl: indexerURL('mainnet')
+  },
+  {
+    ...networks[ChainId.POLYGON],
+    rpcUrl: nodesURL('polygon'),
+    relayer: { url: relayerURL('polygon') },
+    indexerUrl: indexerURL('polygon'),
+    isDefaultChain: true,
+    isAuthChain: true
```

Comment:
Don't know if we want to change the default network from Ethereum to Polygon yet. Can we maintain compatibility with the current release?

Reply:
this one as well is configured this way in webapp

https://github.com/0xsequence/wallet-webapp/blob/master/src/stores/NetworkStore.ts#L52